modules/ROOT/pages/prepare-raml-task.adoc

```diff
@@ -26,15 +27,35 @@ And then you can apply this trait in each individual operation like this:
     description: Gets a list of all the inventory products.
 ----
 
-. Click Policies on the API dashboard.
+== Getting Policies Specification Snippets
+
+. Click *Policies* on the API dashboard.
 +
-In the list of policies, the RAML snippet link contains the RAML code you need to add to the RAML.
+In the list of policies, the RAML snippet link contains the RAML trait or OAS components you need to add to the API specification.
 +
 image::raml-snippet.png[]
-. Click RAML Snippet.
+. Click *API Specification Snippet*.
+. Select the API type to match your specification:
+** RAML 0.8 or RAML 1.0 to match the API RAML version.
 +
 image::snippet-rate-limiting.png[]
-. Select RAML 0.8 or RAML 1.0 to match the API RAML version.
+** Or OAS 2.0 to match the API OAS version.
++
+image::prepare-raml-task-2b676.png[]
++
+OAS Snippets is not supported by all policies versions. +
+See the table below to see the supported policies and versions:
++
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Policy | Version
+| Rate limiting - SLA based | since v1.1.2
+| OAuth 2.0 access token enforcement using Mule OAuth provider | since v1.1.3
+| OpenAM access token enforcement | since v1.1.3
+| OpenId Connect access token enforcement | since v1.1.3
+| OpenId Connect access token enforcement | since v1.1.3
+| Client ID enforcement | since v1.1.3
+|===
 . Edit the API to add the code.
```

Comment:
@FedeAmdan can you review that the versions are correct?

Reply:
@fermujica The versions are correct, but "OpenId Connect access token enforcement" is repeated. The second time should say: "PingFederate access token enforcement policy "

Reply:
Fixed in 44b797f